permission_handler/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 8.1.4+1
+
+*Fix a bug where after allowing the `location` permission and requesting it again would lead to a memory error.
```

Comment:
nit: missing space, if this is not correct it will not be parsed correctly on pub.dev:
```suggestion
* Fix a bug where after allowing the `location` permission and requesting it again would lead to a memory error.
```